src/tools/pub/pubspec.md

```diff
@@ -269,6 +273,65 @@ For more information, see
 [pub global](/tools/pub/cmd/pub-global#running-a-script-from-your-path).
 
 
+### Platforms
+
+When publishing packages to the [pub.dev site.]({{site.pub}}) the set of
+_supported platforms_ will be automatically detected based on transitively
+imported `dart:...` libraries and [Flutter plugin declarations][]. The set of
+supported platforms is displayed on the package page, and used for filtering
+when searching for packages on the [pub.dev site.]({{site.pub}}).
+
+If the supported platforms automatically detected is incorrect, the list of
+_supported platforms_ can be explicitly declared using the `platforms:` key
+as follows:
+
+{% prettify yaml tag=pre+code %}
+# This package supports all platforms listed below
+platforms:
+  android:
+  ios:
+  linux:
+  macos:
+  web:
+  windows:
+{% endprettify %}
```

Comment:
````suggestion
```
# This package supports all platforms listed below
platforms:
  android:
  ios:
  linux:
  macos:
  web:
  windows:
```
````